data/pathfinder/paizo/roleplaying_game/occult_adventures/_occult_adventures.pcc

```diff
@@ -56,18 +55,19 @@ TEMPLATE:oa_templates.lst
 #SPELL:occult_rituals.lst
 
 # Support
-ABILITY:support/oa_feats_uc.lst|PRECAMPAIGN:1,INCLUDES=Ultimate Combat
-ABILITY:support/oa_feats_arg.lst|PRECAMPAIGN:1,INCLUDES=Advanced Race Guide
 SPELL:support/oa_spells_um.lst|PRECAMPAIGN:1,INCLUDES=Ultimate Magic
+SPELL:support/oa_spells_acg.lst|PRECAMPAIGN:1,INCLUDES=Advanced Class Guide
+ABILITY:support/oa_feats_uc.lst|PRECAMPAIGN:1,INCLUDES=Ultimate Combat
 SPELL:support/oa_spells_uc.lst|PRECAMPAIGN:1,INCLUDES=Ultimate Combat
+
+ABILITY:support/oa_feats_arg.lst|PRECAMPAIGN:1,INCLUDES=Advanced Race Guide
 SPELL:support/oa_spells_arg.lst|PRECAMPAIGN:1,INCLUDES=Advanced Race Guide
-SPELL:support/oa_spells_acg.lst|PRECAMPAIGN:1,INCLUDES=Advanced Class Guide
 SPELL:support/oa_spells_ma.lst|PRECAMPAIGN:1,INCLUDES=Mythic Adventures
 SPELL:support/oa_spells_mc.lst|PRECAMPAIGN:1,INCLUDES=Monster Codex
 
 # Needed for a Medium archetype
-ABILITY:@/pathfinder/paizo/roleplaying_game/bestiary_3/b3_abilities_race.lst|(INCLUDE:CATEGORY=Internal,Race Traits ~ Kami (Shikigami)|CATEGORY=Special Ability,Shikigami ~ Improvised Weapon Mastery|CATEGORY=Special Ability,Shikigami ~ Spell-Like Abilities)|!PRECAMPAIGN:1,Bestiary 3
-RACE:@/pathfinder/paizo/roleplaying_game/bestiary_3/b3_races.lst|(INCLUDE:Kami (Shikigami))|!PRECAMPAIGN:1,Bestiary 3
+ABILITY:@/pathfinder/paizo/roleplaying_game/bestiary_3/b3_abilities_race_oa.lst|!PRECAMPAIGN:1,Bestiary 3
+RACE:@/pathfinder/paizo/roleplaying_game/bestiary_3/b3_races_oa.lst|(INCLUDE:Kami (Shikigami))|!PRECAMPAIGN:1,Bestiary 3
 
```

Comment:
The OA extensions should be in a /support folder of the OA folder, and load these as set up. We don't alter the original books when possible.